chroma-manager/tests/integration/utils/test_blockdevices/test_blockdevice_linux.py

```diff
@@ -20,6 +20,10 @@ def preferred_fstype(self):
     def device_path(self):
         return self._device_path
 
+    @property
+    def wipe_device_commands(self):
+        return ['wipefs -a {}'.format(self.device_path)]
```

Comment:
the difference is that with ldiskfs, self.device_path is always equal to self._device_path therefore we don't have to call the private variable _device_path. so whilst it may be an equivalent call this makes that explicit. I can change it if you feel strongly about it.